invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemMetadata/ItemMetadata.js

```diff
@@ -8,42 +11,36 @@ import {
   Table,
   Icon,
 } from 'semantic-ui-react';
-import PropTypes from 'prop-types';
-
 import { CreateNewLoanModal } from './components/CreateNewLoanModal';
+import { invenioConfig } from '../../../../../common/config';
 
 import './ItemMetadata.scss';
-import { invenioConfig } from '../../../../../common/config';
 
 export default class ItemMetadata extends Component {
-  openEditor(url) {
-    window.open(`/editor?url=${url}`, url);
-  }
-
   render() {
-    const data = this.props.itemDetails.metadata;
-
-    const itemUrl = this.props.itemDetails.links.self;
+    const { item } = this.props;
     return (
       <Segment className="item-metadata">
         <Grid padded columns={2}>
           <Grid.Column width={16}>
             <Header as="h1">
-              Item - {data.barcode}
+              Item - {item.metadata.barcode}
               <Button
                 primary
                 floated="right"
                 size="small"
-                onClick={() => this.openEditor(itemUrl)}
+                onClick={() =>
+                  window.open(`/editor?url=${item.links.self}`, item.links.self)
```

Comment:
I think this is normally blocked by browsers. Better replacing it with:
`<a href="" target="_blank">`